### ISSUE TITLE
Slider Syncing + slickFilter

### ISSUE THREAD

**Issue by morinos** [kenwheeler/slick#3833]
### Description:

Synchronized sliders are getting desync when using the slickFilter method
====================================================================

https://jsfiddle.net/w0o4mr7k/8/

====================================================================

#### Steps to reproduce the problem

1. Click on the "Filter green slides" button 
2. Navigate through the slides using the small thumbnails
3. It should be desynchronized

====================================================================

#### What is the expected behaviour?

After filtering slides, you should be able to navigate smoothly using the thumbnail slider

====================================================================

#### What is observed behaviour?

Slides are not being re-indexed, I'm using a similar setup on a website and I only have this problem when the slidesToShow is higher or equal to the number of filtered slides.

====================================================================

#### More Details

- Chrome (Latest version)
- Can't specify the slick version used on the website because someone removed the comments in the js file.
- Did this work before? No

**Note:** it's my first ever post on Github so I apologize if I did it wrong.

**Comment by andrewcbi** [comment]
yes have found this to happen also, my way around it was to make the swatches of the main slider directly call the other slider underneath when clicked on using the gotoslide function. It is a bit to keep track of but if not changing too often it works.

**Comment by artemas** [comment]
I've got a similar problem with 1.6.0, upgrade to 1.8.1 doesn't help.

**Comment by jinch** [comment]
Me too : / ~ has there been any resolution for this?

**Comment by diff-michael-m** [comment]
> Me too : / ~ has there been any resolution for this?
Nope sadly

**Comment by jinch** [comment]
It looks like the classes on the thumbnails are not being properly applies. I have noticed that these 
".slick-current .slick-active .slick-center" do not always translate when filter is applied.
